assets/js/components/ExecutionResults/CheckResultOutline.jsx

```diff
@@ -36,17 +44,49 @@ function CheckResultOutline({
         isAgentCheckError || metExpectations < expectExpectationsCount,
     };
   });
+};
+
+const expectSameExpectations = (expectationList) =>
+  expectationList.filter(isExpectSame);
+
+const getExpectSameExpectationResult = (expectationResults, name) =>
+  expectSameExpectations(expectationResults).find(
+    ({ name: resultExpectationName }) => name === resultExpectationName
+  ) || {};
```

Comment:
You can achieve the same result using an `if` statement

Reply:
Yes I know, and if it helps you better, the best I can do is change it to
```js
const getExpectSameStatementResult = (expectationResults, name) => {
  const expectSameStatement = getExpectSameStatements(expectationResults).find(
    ({ name: resultExpectationName }) => name === resultExpectationName
  );

  if (!expectSameStatement) {
    return {};
  }

  return expectSameStatement;
};
```